pipeline.jsonc

```diff
@@ -1,8 +1,8 @@
 {
     "eosio": {
         "environment": {
-            "IMAGE_TAG": "1.8.0-rc2",
-            "HELPER_ENABLED": true
+            "HELPER_ENABLED": true,
```

Comment:
Blockchain has asked us to use the `pipeline.jsonc` file only when we need to override default pipeline behavior, but `HELPER_ENABLED='true'` is required for the pipeline to function and so should be implemented in one of the following places:
- Pipeline upload script
- Pipeline YAML
- Pipeline terraform

As such, it should be deleted here.